cmd/otel-allocator/internal/config/config.go

```diff
@@ -48,6 +49,12 @@ var (
 	DefaultKubeConfigFilePath string = filepath.Join(homedir.HomeDir(), ".kube", "config")
 )
 
+var defaultScrapeProtocolsCR = []monitoringv1.ScrapeProtocol{
```

Comment:
do we want this to land after #3998 ?

Reply:
If we do, I will update :)

Reply:
Alternative, can land this first (since this not change defaults) and update the defaults for this path as well in the other PR.

Reply:
lets do the other one first and then this one, i think it'll make this one even more clear

Reply:
Ready to merge.